.eslintrc.cjs

```diff
@@ -268,6 +268,12 @@ const overrides = [
       'rulesdir/cactbot-trigger-property-order': 'error',
```

Comment:
It is a little weird that the raidboss one applies to both. Can this be pulled out into a separate raidboss-only section, module: raidboss, as a peer to oopsyraidsy directly below.